### PR TITLE
AM-3002 - Introduced iteration rounds for SHA password encoding

### PR DESCRIPTION
**Pull request by lgw-gravitee**

### CODE REVIEW

**Review by podlesrafal — APPROVED** [review]

File: gravitee-am-service/src/test/java/io/gravitee/am/service/authentication/crypto/password/SHAMD5PasswordEncoderTest.java [hunk 1]
```diff
@@ -33,17 +36,21 @@ protected PasswordEncoder getEncoder() {
         return this.messageDigestPasswordEncoder;
     }
 
-    @Test
```
Comment (mstasgravitee) [inline gravitee-am-service/src/test/java/io/gravitee/am/service/authentication/crypto/password/SHAMD5PasswordEncoderTest.java:36]: I would suggest adding a test with a negative round

**Review by mstasgravitee — APPROVED** [review]
